lib/doc2pdf/document.rb

```diff
@@ -1,14 +1,17 @@
 # frozen_string_literal: true
 
 require "docx"
+require "uri"
 
 module Doc2pdf
-  # Wraps a Docx::Document.
+  # Wraps a ::Docx::Document.
   class Document
-    attr_reader :docx
+    attr_accessor :docx
```

Comment:
Do you actually need the setter method `#docx=` to be public?

Reply:
I did commit by mistake some test changes